hosting/src/app/(protected)/content/article/page.tsx

```diff
@@ -4,24 +4,50 @@ import Loader from "@/components/common/Loader";
 import Notification from "@/components/common/Notification";
 import PageHeader from "@/components/common/PageHeader";
 import {useTanamDocumentType} from "@/hooks/useTanamDocumentTypes";
-import {useTanamDocuments} from "@/hooks/useTanamDocuments";
+import {useCreateTanamDocument, useTanamDocuments} from "@/hooks/useTanamDocuments";
 import {UserNotification} from "@/models/UserNotification";
+import {useRouter} from "next/navigation";
 import {Suspense, useEffect, useState} from "react";
 
 export default function DocumentTypeDocumentsPage() {
   const {data: documentType} = useTanamDocumentType("article");
+  const {create, error: writeError, isLoading} = useCreateTanamDocument(documentType?.id);
   const {data: documents, error: docsError} = useTanamDocuments("article");
   const [notification, setNotification] = useState<UserNotification | null>(null);
+  const router = useRouter();
 
   useEffect(() => {
-    setNotification(docsError);
-  }, [docsError]);
+    setNotification(docsError || writeError);
+  }, [docsError, writeError]);
+
+  const addNewArticle = async () => {
+    const result = await create();
+
+    router.push(`article/${result?.id}`);
+  };
 
   return (
     <>
       <Suspense fallback={<Loader />}>
-        {documentType ? <PageHeader pageName={documentType.titlePlural.translated} /> : <Loader />}
+        {documentType ? (
+          <div className="flex items-center">
+            <PageHeader pageName={documentType.titlePlural.translated} />
+            <button
+              type="button"
+              className="ml-4 mb-6 inline-flex items-center justify-center rounded-md bg-primary px-2 py-2 text-center font-medium text-white hover:bg-opacity-90 lg:px-2 xl:px-2"
+              onClick={addNewArticle}
+            >
```

Comment:
Use existing button component for this: https://github.com/oddbit/tanam/blob/main/hosting/src/components/Button.tsx